src/config/modelProviders/groq.ts

```diff
@@ -3,6 +3,27 @@ import { ModelProviderCard } from '@/types/llm';
 // ref https://console.groq.com/docs/models
 const Groq: ModelProviderCard = {
   chatModels: [
+    {
+      displayName: 'LLaMA3.1 405B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-405b-reasoning',
+      tokens: 131_072,
+    },
+    {
+      displayName: 'LLaMA3.1 70B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-70b-versatile',
+      tokens: 131_072,
+    },
+    {
+      displayName: 'LLaMA3.1 8B',
+      enabled: true,
+      functionCall: true,
+      id: 'llama-3.1-8b-instant',
+      tokens: 131_072,
+    },
```

Comment:
Since these models are in Preview state, it is recommended to add Preview to their `displayName`.